### ISSUE TITLE
Atem Extreme pro iso won't connect to companion app

### ISSUE THREAD

**Issue by SeesSeven** [bitfocus/companion-module-bmd-atem#259]
I'm trying to control may extreme iso pro with my stream deck XL 2. I downloaded a profile only from David Joshua Ford's website. I've matched up the IP address but the status only says connecting but never connects. The icons for the companion profile I downloaded for show up on the stream decks buttons, but it won't control my switcher.

Anyone know what I need too do to trouble shoot getting my switcher to communicate with my stream deck through companion??

**Comment by davidjoshuaford** [comment]
Hi SeesSeven, what version of Companion, and what ATEM firmware version are you running?

**Comment by dickmann** [comment]
Hi @davidjoshuaford: Are you using Companion Pi with release 3.1. or later with your switcher profiles, is the connection working? I have the same issue as SeesSeven, see my new issue I opened today in parallel....

**Comment by davidjoshuaford** [comment]
Hi @dickmann I haven't used Companion Pi in the last several months, but I have just run a test on M1 Macbook Pro importing the profile to stable version 3.0.1 and the latest beta and they both work.

**Comment by pinnaql3** [comment]
I'm having the same issue. Also using David Joshua Ford's stream deck profile. Atem is no longer connecting in Companion.

**Comment by davidjoshuaford** [comment]
@pinnaql3 if you start a fresh connection within v3.1.2+ of Companion, is that able to connect to the ATEM? 

Trying to rule out if there's an incompatibility of a v2 config file importing into v3.

**Comment by pinnaql3** [comment]
Hi, David.

So far, no.
Even with a fresh connection, it still won't connect.

**Comment by davidjoshuaford** [comment]
Sounds like it might be something to do with the network/BMD rather than Companion. I've had times where my SDI Extreme ISO doesn't appear on the network and I can't figure it out and power cycling it has helped. 

Also check if DHCP is on, that there aren't any other devices using the same IP address on the network that may be blocking each other.

**Comment by pinnaql3** [comment]
Thanks for your help.

It was a networking issue on my end. I was able to go back and utilize both
Companion and your all of your profiles.

**Comment by davidjoshuaford** [comment]
Glad you got it sorted!
